Cannot find module './package.json'

**_markbind/logs/markbind-2018-04-22.log**:

```
2018-04-22T10:19:13.854Z - error: uncaughtException: Cannot find module './package.json' date=Sun Apr 22 2018 18:19:13 GMT+0800 (Malay Peninsula Standard Time), pid=6568, uid=null, gid=null, cwd=C:\Users\Aaron\Desktop\tmp, execPath=C:\Program Files\nodejs\node.exe, version=v8.0.0, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Aaron\AppData\Roaming\npm\node_modules\markbind-cli\index.js, init], rss=78008320, heapTotal=65675264, heapUsed=33215936, external=156116, loadavg=[0, 0, 0], uptime=486.3648792
Error: Cannot find module './package.json'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\Aaron\AppData\Roaming\npm\node_modules\markbind-cli\lib\markbind\node_modules\cheerio\index.js:11:19)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\Aaron\AppData\Roaming\npm\node_modules\markbind-cli\lib\markbind\lib\parser.js:16:17)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
```

--- a/.npmignore
+++ b/.npmignore
@@ -1,1 +1,4 @@
-src/
+__mocks__
+docs
+node_modules
+test

